bootstrap/mocks/things.go

```diff
@@ -51,6 +51,25 @@ func (svc *mainfluxThings) AddThing(_ context.Context, owner string, thing thing
 	return thing, nil
 }
 
+func (svc *mainfluxThings) CreateThings(_ context.Context, owner string, ths []things.Thing) ([]things.Thing, error) {
```

Comment:
@manuio `things` are import, then this would have to be renamed, and I do not like this.

I also feel that `ths` would be more Go idiomatic in this case. Let's keep it as it is.

Reply:
It's not consistent with `for i := range channels {`. I would use `t` and `c` if you want to be more Go idiomatic

Reply:
@manuio `ths` and `chs` would be better fit then IMHO, as this refers to an array, several things, not just one.